bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/i18n/LocaleProvider.java

```diff
@@ -25,4 +28,10 @@
      */
     Locale getLocale();
 
+    /**
+     * Provides access to the location of the installation
+     *
+     * @return location of the current installation or null if the location is not set
+     */
+    PointType getLocation();
```

Comment:
@martinvw I agree that `i18n` might be a bit misleading but our idea was to put it into the `LocaleProvider` so if the user sets up a location but no `locale` we could derive a `locale` based on the position. Such a logic can be implemented a lot easier than if we separate them into two services.